scripts/apps/authoring/views/article-edit.html

```diff
@@ -641,7 +641,7 @@
         data-single-line="field.field_options.single"
         data-value="item.extra[field._id]"
         data-bind-to-value="compareView"
-        data-scroll-container=".page-content-container"
+        data-scroll-container=".ppage-content-container--scrollable"
```

Comment:
seems like typo `.PPage`

Reply:
here's the fix https://github.com/superdesk/superdesk-client-core/pull/3867